tests/wallet/test_wallet_hd.py

```diff
@@ -42,7 +43,8 @@ def test_transaction_and_balance(self):
         out = WalletOutputInfo(decode_address(new_address2), self.TOKENS, timelock=None)
         tx1 = self.wallet.prepare_transaction_compute_inputs(Transaction, [out], self.tx_storage)
         tx1.update_hash()
-        tx1.verify_script(tx1.inputs[0], block)
+        verifier = TransactionVerifier(settings=self._settings)
```

Comment:
Why not `self.manager...`?

Reply:
Indeed. Changed in https://github.com/HathorNetwork/hathor-core/pull/800/commits/bc5f26335bd22d387867e6e3c5a77cb556af4cca

Reply:
Thanks!